plugins/ftdidmx/FtdiWidget.cpp

```diff
@@ -127,7 +136,11 @@ void FtdiWidget::Widgets(vector<FtdiWidgetInfo> *widgets) {
       ftdi_device_list* current_device = list;
 
       while (current_device != NULL) {
+#if HAVE_LIBFTDI1
+        struct libusb_device *dev = current_device->dev;
+#else
         struct usb_device *dev = current_device->dev;
+#endif //HAVE_LIBFTDI1
```

Comment:
Can you fix the lint error here:
https://travis-ci.org/OpenLightingProject/ola/jobs/384567230#L1754

Reply:
Fixed